code/modules/jobs/job_types/followers.dm

```diff
@@ -345,24 +345,56 @@ Follower Volunteer
 		/obj/item/flashlight/seclite = 1,
 		/obj/item/storage/firstaid/ancient = 1,
 		/obj/item/stock_parts/cell/ammo/ec = 2,
+		/obj/item/melee/onehanded/knife/hunting=1,
+	)
+
+/datum/outfit/job/followers/f13followerscholar/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	ADD_TRAIT(H, TRAIT_TECHNOPHREAK, src)
+	ADD_TRAIT(H, TRAIT_RESEARCHER, src)
+	ADD_TRAIT(H, TRAIT_CYBERNETICIST_EXPERT, src)
+	ADD_TRAIT(H, TRAIT_CYBERNETICIST, src)
+
+/datum/outfit/loadout/engineer //Upgraded tools, RCD
+	name = "Followers Engineer"
+	belt = /obj/item/storage/belt/utility/full/engi
+	head =	/obj/item/clothing/head/hardhat
+	gloves =	/obj/item/clothing/gloves/color/yellow
+	backpack_contents = list(
+		/obj/item/construction/rcd = 1,
+		/obj/item/lightreplacer = 1,
+		/obj/item/rcd_ammo = 4,
 	)
 
-/datum/outfit/loadout/guard_ranged
-	name = "Followers Ranged Guard"
-	suit_store = /obj/item/gun/ballistic/rifle/repeater/cowboy
+/datum/outfit/loadout/miner //Quick access to mining, scanner
+	name = "Followers Miner"
+	mask = /obj/item/clothing/mask/gas/explorer
+	head =	/obj/item/clothing/head/hardhat/orange
 	backpack_contents = list(
-		/obj/item/ammo_box/tube/a357 = 2,
+		/obj/item/storage/bag/ore/large = 2,
+		/obj/item/pickaxe/titanium = 1,
+		/obj/item/t_scanner/adv_mining_scanner = 1,
 	)
 
-/datum/outfit/loadout/guard_close
-	name = "Followers Close Protection Guard"
-	suit_store = /obj/item/gun/ballistic/shotgun/hunting
+/datum/outfit/loadout/combat //Guard, but actually good.
+	name = "Followers Crusader"
+	suit = /obj/item/clothing/suit/hooded/followerheavy
+	belt = /obj/item/storage/belt/military/army/military/
 	backpack_contents = list(
+		/obj/item/gun/energy/laser/wattz/recharger = 1,
+		/obj/item/gun/ballistic/shotgun/police = 1,
```

Comment:
Could give them the recharger rifle, no?

Reply:
IS there a recharger rifle?